Forge.TreeWalker/src/TreeWalkerParameters.cs

```diff
@@ -180,8 +192,48 @@ public class TreeWalkerParameters
             this.SessionId = sessionId;
             this.JsonSchema = jsonSchema;
             this.ForgeState = forgeState;
+            this.Token = token;
             this.Callbacks = callbacks;
```

Comment:
Flip the order here. Callbacks before token to line up with parameters.

Reply:
updated

---
In reply to: [506636201](https://github.com/microsoft/Forge/pull/32#discussion_r506636201) [](ancestors = 506636201)